src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -410,4 +410,14 @@ public function testNotSupportedMissingValue()
     {
         Inline::parse('{this, is not, supported}');
     }
+
+    public function testVeryLongQuotedStrings()
+    {
+        $longStringWithQuotes = str_repeat("x\r\n\\\"x\"x", 1000);
+
+        $yamlString = Inline::dump(['longStringWithQuotes' => $longStringWithQuotes]);
```

Comment:
the short array syntax does not work on PHP 5.3, which is still supported by Symfony 2.7. Please change it.